[TS] typedoc TS1479 error

```javascript
node_modules/antd/es/config-provider/context.d.ts:3:30 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("scroll-into-view-if-needed")' call instead.

3 import type { Options } from 'scroll-into-view-if-needed';
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/antd/es/config-provider/index.d.ts:3:30 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("scroll-into-view-if-needed")' call instead.

3 import type { Options } from 'scroll-into-view-if-needed';
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/antd/es/form/Form.d.ts:4:30 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("scroll-into-view-if-needed")' call instead.

4 import type { Options } from 'scroll-into-view-if-needed';
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/antd/es/form/interface.d.ts:2:47 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("scroll-into-view-if-needed")' call instead.

2 export type { Options as ScrollOptions } from 'scroll-into-view-if-needed';
                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "node16",
     "declaration": true,
     "rootDir": "src",
     "outDir": "dist",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-    "moduleResolution": "node16",
+    "moduleResolution": "nodenext",
     "declaration": true,
     "rootDir": "src",
     "outDir": "dist",

